src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Extensions/JsRuntimeExtension.cs

```diff
@@ -8,5 +8,10 @@ public static async Task SetToggleBodyOverflow(this IJSRuntime jsRuntime, bool i
         {
             await jsRuntime.InvokeVoidAsync("toggleBodyOverflow", isNavOpen);
         }
+
+        public static async Task ScrollToFragmentOnClickEvent(this IJSRuntime jsRuntime, string targetElementId)
+        {
+            await jsRuntime.InvokeVoidAsync("scrollToFragmentOnClickEvent", targetElementId);
```

Comment:
please rename these methods in both C# and JS to a proper name like `ScrollToElement`